### ISSUE TITLE
minibuffer-keyboard-quit not defined

### ISSUE THREAD

**Issue by josh-nz** [mbriggs/old-emacs.d#2]
I'm trying to use your escape key mappings for evil mode as per http://stackoverflow.com/questions/8483182/emacs-evil-mode-best-practice and have some issues. If I press meta-x then escape, I get an error 'Symbol's function definition is void: minibuffer-keyboard-quit'.

Any ideas? I'm running emacs 24.3.

**Comment by gsastry** [comment]
Also getting this error, did you resolve this josh?

**Comment by josh-nz** [comment]
Possibly. I didn't get many results searching for minibuffer-keyboard-quit in Google when I first has this issue. Had another quick try tonight, and found reference to a custom function https://github.com/davvil/.emacs.d/blob/master/init.el which may be the missing link. I've grep'd this repo and it's not present. Presumably mbriggs has the same issue, although he answered the Stack Overflow question I linked to above, so perhaps his function is simply missing from this repo. Unsure whether mbriggs's function is the same as the one I just linked to.
